src/import/v8cpuFormatter.ts

```diff
@@ -0,0 +1,61 @@
+import {CPUProfile, CPUProfileNode} from './chrome'
+
+export interface OldCPUProfileNode {
+  functionName: string
+  lineNumber: number
+  scriptId: string
+  url: string
+  hitCount: number
+  bailoutReason: string
+  id: number
+  children: OldCPUProfileNode[]
+}
+
+export interface OldCPUProfile {
+  startTime: number
+  endTime: number
+  head: OldCPUProfileNode
+  samples: number[]
+  timestamps: number[]
+}
+
+function treeToArray(node: OldCPUProfileNode, nodes: CPUProfileNode[]) {
+  nodes.push({
+    id: node.id,
+    callFrame: {
+      columnNumber: 0,
+      functionName: node.functionName,
+      lineNumber: node.lineNumber,
+      scriptId: node.scriptId,
+      url: node.url,
+    },
+    hitCount: node.hitCount,
+    children: node.children.map(child => child.id),
+  })
+  node.children.forEach(child => {
+    return treeToArray(child, nodes)
+  })
+  return nodes
+}
+
+function timestampsToDelta(timestamps: number[], startTime: number): number[] {
+  return timestamps.reduce((deltas: number[], timestamp: number, index: number) => {
+    let lastTimestamp = index === 0 ? startTime * 1000000 : timestamps[index - 1]
+    deltas.push(timestamp - lastTimestamp)
+    return deltas
+  }, [])
+}
+
+/**
+ * Convert the tree based format to the array of nodes like actually
+ */
+export function chromeTree2nodes(content: OldCPUProfile): CPUProfile {
+  // Care that both startTime and endTime are now in microsecond
```

Comment:
I think the more natural way of saying this would be:

```suggestion
  // Note that both startTime and endTime are now in microseconds
```

How would you say this in French? Attention?

Reply:
Yes, somethings among the lines of `Attention, les deux variables startTime et endTime sont en microseconds`.